index.html

```diff
@@ -317,41 +484,47 @@ <h2>Skills.</h2>
     </section>
 
     <section id="form-container">
-      <div>
-        <h2>Contact me</h2>
-        <p>
-          If you have an application you are interested in developing, a feature
-          that you need built or a project that needs coding. I’d love to help
-          with it!
-        </p>
-      <form 
-        action="https://formspree.io/f/mdovrprr"
-        method="POST">
-        <ul>
-          <li>
-            <input type="text" maxlength="30"  placeholder="Lorem Ipsum" required />
-          </li>
-          <li>
-            <input type="email" placeholder="loremipsum@gmail.com" required />
-          </li>
-          <li>
-            <textarea
-              name="message"
-              cols="28"
-              rows="7"
-              maxlength="500"
-              required
-              placeholder="Write you message here..."
-            ></textarea>
-          </li>
-          <li>
-            <button type="submit">Get in touch</button>
-          </li>
-        </ul>
-      </form>
-      <img src="./assets/Shape.png" alt="bar">
+      <div id="contact-form-bg-container">
+        <div id="contact-form-wrapper">
+          <h2>Contact me</h2>
+          <p>
```

Comment:
- [ ] **[OPTIONAL]** You've done an incredible job on your entire portfolio! I have a suggestion to enhance its appearance and make it look even more professional. I noticed that there's a blank space around the borders of your entire page. To address this, I recommend adding a `CSS` reset to your `CSS` file. Here's an example of how you can achieve it:

   ```
    * {
     padding: 0;
     margin: 0;
    }
   ```

   By including this code snippet, will ensure that the `padding` and `margin` properties are reset to zero for all elements on your page. This **will help eliminate any unwanted spaces and create a cleaner and more polished look.** 🙏
   
   **_You can read more about CSS resets in this [useful link](https://itnext.io/basic-css-resets-to-apply-in-every-app-9bd724dc4f49)._**

   ### Blank space:
   ![image](https://github.com/CobbyElsonfx/Portfolio-Website/assets/107952724/fe3148c8-9e82-40fa-9176-1c6ba5d20ab6)